### DISCUSSION TITLE
Code Server's Terminal user and password?

### DISCUSSION THREAD

**Discussion by PatrickHennecke** [coder/code-server#6054] [Q&A]
Hello,
Forgive my lack of knowledge. I'm unsure when and where the user for the terminal was set - I'm not sure what password to use with this user 'abc' that I have. For reference, I'm running the code server on a Synology box. I attempted reseting the password using "passwd" and "sudo passwd abc" however, I am just asked for the password for user 'abc'. Any help would be greatly appreciated. I was using linuxserver/code-server for my docker container... I switched to codercom/code-server just now. I'll update if the problem persists or not.

**Comment by PatrickHennecke** [comment]
Transferring to codercom/code-server fixed the issue!

**Comment by wanghan17** [comment]
so am i
i just dont know what is the password

**Reply by code-asher** [reply]
`codercom/code-server` has passwordless sudo.  For `linuxserver/code-server` you can ask here: https://github.com/linuxserver/docker-code-server
